action_tutorials/action_tutorials_cpp/src/fibonacci_action_server.cpp

```diff
@@ -73,11 +73,73 @@ class FibonacciActionServer : public rclcpp::Node
       handle_goal,
       handle_cancel,
       handle_accepted);
+
+    auto on_set_parameter_callback =
+      [](std::vector<rclcpp::Parameter> parameters) {
```

Comment:
include `<vector>`

Reply:
Thanks. Fix it at https://github.com/ros2/demos/pull/709/commits/ffc65d6c050e4550e4287baa6ebad0b05b76e9de